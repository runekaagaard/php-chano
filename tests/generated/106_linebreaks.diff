001- <p>123</p>
001+ Fatal error: Call to undefined method DtlIter::linebreaks() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/106_linebreaks.php on line 4