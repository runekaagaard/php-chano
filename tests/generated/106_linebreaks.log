
---- EXPECTED OUTPUT
<p>123</p>
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::linebreaks() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/106_linebreaks.php on line 4
---- FAILED
