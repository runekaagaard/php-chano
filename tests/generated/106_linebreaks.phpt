--TEST--
A generated testfile for the "linebreaks" filter.
--FILE--
<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 123)));
foreach ($items as $i) echo $i->input->linebreaks();
--EXPECT--
<p>123</p>