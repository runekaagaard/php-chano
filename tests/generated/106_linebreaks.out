Fatal error: Call to undefined method DtlIter::linebreaks() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/106_linebreaks.php on line 4